Modules/Brand/Http/Services/BrandService.php

```diff
@@ -0,0 +1,509 @@
+<?php
+
+namespace Modules\Brand\Http\Services;
+
+use Illuminate\Support\Str;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rules\Password;
+use Modules\Brand\Entities\Catalog;
+use Modules\Country\Entities\Country;
+use Modules\User\Entities\User;
+use Modules\Brand\Entities\Brand;
+use File;
```

Comment:
I think this should be Illuminate\Support\Facades\File;

Reply:
done